README.rst

```diff
@@ -86,20 +79,20 @@ Requirements
   .. admonition:: Binary filename suffix
 
      The created binaries have an ``.exe`` suffix on Windows. On other platforms
-     they have no suffix for standalone mode, or ``.bin`` suffix, that you ar
+     they have no suffix for standalone mode, or ``.bin`` suffix, that you are
      free to remove or change, or specify with the ``-o`` option.
 
      The suffix for acceleration mode is added just to be sure that the original
      script name and the binary name do not ever collide, so we can safely do
      an overwrite without destroying the original source file.
 
-  .. admonition:: It **has to** be CPython, AnaConda or MiniConda Python.
```

Comment:
I see, I might have used that spelling in a couple of places, now I realize it's only in my head, no idea why. :-)

Reply:
At first glance, even I thought it was right 😃